Access Medicine.js

```diff
@@ -0,0 +1,264 @@
+{
+	"translatorID": "60e55b65-08cb-4a8f-8a61-c36338ec8754",
+	"label": "Access Medicine",
+	"creator": "Jaret M. Karnuta",
+	"target": "^https?://(0-)?accessmedicine\\.mhmedical\\.com",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-12-26 19:52:12"
+}
+
+function detectWeb(doc, url){
+	var citation = doc.getElementById("hfGetCitation");
+	if(citation){
+		return "bookSection";
+	}
+}
+
+function doWeb(doc, url){
+	var contentType = detectWeb(doc, url);
+	if (contentType != "bookSection"){return;}
+
+	//only book section from now on
+	var href = doc.location.href.split("/content")[0];
+	var sectionId = doc.location.href.split("sectionid=")[1];
+	var link = href + "/downloadCitation.aspx?format=ris&sectionid=" + sectionId;
+
+	var chapterSpan = doc.getElementById('pageContent_lblChapterTitle1');
+	Zotero.Utilities.doGet(link, function(risText){
+		//set RIS import translator
+		var translator = Zotero.loadTranslator("import");
+		translator.setTranslator("32d59d2d-b65a-4da4-b0a3-bdd3cfb979e7");
+		translator.setString(risText);
+		translator.setHandler("itemDone", function(obj, item) {
+			if (chapterSpan) {
+				//remove 'chapter' text if present
+				var chapterText = chapterSpan.innerHTML.toLowerCase().replace("chapter",'').trim();
+				item.notes.push('Chapter: '+chapterText);
+			}
+			item.attachments.push({
+				title: "Snapshot",
+				document: doc
+			});
+			item.complete();
+		})
+		translator.translate();
+	});
+}
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://accessmedicine.mhmedical.com.ccmain.ohionet.org/content.aspx?bookid=1568&sectionid=95701373",
+		"items": [
+			{
+				"itemType": "bookSection",
+				"title": "Sympathomimetics",
+				"creators": [
+					{
+						"lastName": "Trevor",
+						"firstName": "Anthony J.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Katzung",
+						"firstName": "Bertram G.",
+						"creatorType": "author"
+					},
+					{
+						"lastName": "Kruidering-Hall",
+						"firstName": "Marieke",
+						"creatorType": "author"
+					}
+				],
+				"date": "2015",
+				"abstractNote": "The  sympathomimetics  constitute  a  very  important  group  of  drugs  used  for  cardiovascular,  respiratory,  and  other  conditions.  They  are  readily  divided  into  subgroups  on  the  basis  of  their  spectrum  of  action  (α-,  β-,  or  dopamine-receptor  affinity)  or  mode  of  action  (direct  or  indirect).",
```

Comment:
I would suggest to handle the double spaces here as well, i.e. try to add something like
```javascript
if (item.abstractNote) {
  item.abstractNote = item.abstractNote.replace(/\s\s/g, ' ');
}
```
in the `setHandler` method above.